File: app/src/main/java/com/grace/placessearch/common/app/FavoritePlacesPreferences.java
package com.grace.placessearch.common.app;

/**
 * A local data store equivalent to save favorite venue ids.
 */
public class FavoritePlacesPreferences {

    public String favoriteVenues;

    @Override
    public String toString() {
        return String.format("favoriteVenues: %s", favoriteVenues);
    }
}

File: app/src/main/kotlin/com/grace/placessearch/common/app/FavoritePlacesPreferences.kt
package com.grace.placessearch.common.app

/**
 * A local data store equivalent to save favorite venue ids.
 */
class FavoritePlacesPreferences {

    var favoriteVenues: String? = null

    override fun toString(): String {
        return String.format("favoriteVenues: %s", favoriteVenues)
    }
}
